src/state/slices/accountSpecifiersSlice/selectors.ts

```diff
@@ -8,3 +9,19 @@ export const selectAccountSpecifierStrings = (state: ReduxState) =>
   state.accountSpecifiers.accountSpecifiers.map(accountSpecifier =>
     Object.entries(accountSpecifier)[0].join(':')
   )
+
+// Returns a CAIP2-indexed object with all the `caip2:pubkeyish` accounts for that chainId
+// For most accounts, that's effectively a CAIP19, but not for e.g UTXO chains thus the pubkeyish naming
+// We use this in cosmos plugin to get the pubkey as a CAIP19, without needing to use chain-adapters in components
+// Since the pubkey is already in state
+export const selectPubkeyishByChainId = (state: ReduxState, chainId: CAIP2) =>
```

Comment:
will defer to @cjthompson here - let's all this `selectAccountSpecifier` now until we land on something better, clearly `pubkeyish` flopped

Reply:
Renamed

Reply:
Do we also want to rename references to `pubkeyish` to `accountSpecifier` in the body/comments? IMO we should keep them as it makes it quite clear that it's not necessarily an `accountSpecifier` (until we have some more discussions about this terminology and format, cc. @elmutt @DaoDev44)